### PR TITLE
Use goldmark+v2 version of markdownfmt

### PR DESCRIPTION
**Pull request by karelbilek**
~~Also use v2 in import path~~ not needed

(currently using `replace` in go.mod to replace to my PR, that will change)

### CODE REVIEW

**Comment by karelbilek** [comment]
Now that markdownfmt is updated, I use the correct URL in go.mod/go.sum

File: cmd/root.go [hunk 1]
```diff
@@ -204,27 +232,64 @@ func (ci *codeInj) Parse() *bf.Node {
...
 }
 
-func (ci *codeInj) Inject(n *bf.Node, entering bool) bf.WalkStatus {
+// snippet is split to more siblings
+func (ci *codeInj) IsMatchWithSiblings(n ast.Node) (bool, []byte, []ast.Node) {
+	//tnode, is := n.(*ast.Text)
+	//if !is {
+	//	return false, nil, nil
+	//}
+	//literal1 := tnode.Segment.Value(ci.originSource)
```
Comment (Kunde21) [inline cmd/root.go]: Some commented code still included.  Can we clean that out?

File: cmd/root.go [hunk 2]
```diff
@@ -204,27 +232,64 @@ func (ci *codeInj) Parse() *bf.Node {
...
+			return true, joined, nodes
+		}
+
+		i = i.NextSibling()
+	}
+
+	return false, nil, nil
+}
+
+//func(n Node, entering bool) (WalkStatus, error)
```
Comment (Kunde21) [inline cmd/root.go]: Needed?

File: cmd/root.go [hunk 3]
```diff
@@ -145,42 +160,55 @@ func (ci *codeInj) initConfig() {
...
+func (ci *codeInj) injectCode() error {
+	nodes, err := ci.Parse()
+	if err != nil {
+		return err
+	}
+	err = ast.Walk(nodes, ci.Inject)
+	if err != nil {
+		return err
+	}
+	//nodes.Walk(ci.Inject)
```
Comment (Kunde21) [inline cmd/root.go]: Needed?

**Review by Kunde21 — APPROVED** [review]
Some comment cleanup questions, otherwise looks good

File: cmd/root.go (hunk 1 above)
Comment (karelbilek) [reply cmd/root.go]: thx for catching this, will remove

**Pull request merged**